mmengine/config/__init__.py

```diff
@@ -1,4 +1,8 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .config import Config, ConfigDict, DictAction
+from .lazy import LazyAttr, LazyObject
+from .utils import Transform
```

Comment:
Users do not need to use those classes so they can be removed from this `__init__` file.